src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -312,6 +331,16 @@ protected override Encoding StandardOutputEncoding
 
         protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
```

Comment:
Yes, it seems we'll hit #3859 when SkipCompilerExecution is set. That is, the Vbc task will rename the .pdb if the .pdb has a name other than the default.